components/MW/metgene_metabolite_table/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import { z } from 'zod'
+import { valHooks } from 'jquery'
+
+// A unique name for your data type is used here
+export const MetgeneMetaboliteTable = MetaNode.createData('MetgeneMetaboliteTable')
+  // Human readble descriptors about this node should go here
+  .meta({
+    label: 'Metgene metabolite table',
+    description: 'Metgene metabolite table, rendered',
+  })
+  // this should have a codec which can encode or decode the data type represented by this node
+  //  using zod, a compile-time and runtime type-safe codec can be constructed
+  .codec(z.any())
+  // react component rendering your data goes here
+  .view(data => {
+    const heading1 = "Gene"
+    const heading2 = "KEGG_COMPOUND_ID"
+    const heading3 = "REFMET_NAME"
+    const heading4 = "KEGG_REACTION_ID"
+    const heading5 = "METSTAT_LINK"
+    return (
+      <div>
+        <h2>MetGENE metabolites</h2>
+        <table>
+          <tr>
+            <th>{heading2}</th>
+            <th>{heading3}</th>
+            <th>{heading4}</th>
+            <th>{heading5}</th>
+          </tr>
+          {data.map((val, key) => {
+          var all_rxn_ids = val.KEGG_REACTION_ID
+          var cpd_id = val.KEGG_COMPOUND_ID     
+          var rxn_id_arr = all_rxn_ids.split(", ")
```

Comment:
Please replace `var`s with `const` or  with `let` if you need reassignment. `var` can have odd scoping behaviors

Reply:
Done